src/components/views/auth/Welcome.tsx

```diff
@@ -73,4 +74,8 @@ export default class Welcome extends React.PureComponent<IProps> {
             </AuthPage>
         );
     }
+
+    componentDidMount() {
+        getAnalytics().trackAnonymousEvent<IWelcomeScreenLoad>("welcome_screen_load", {});
```

Comment:
maybe `{}` should be the default value for that param to make these param-less ones a little nicer